### ISSUE TITLE
[SF][ObjectPage][A11y]: Cannot remove the role="banner" from ObjectPageTopHeader

### ISSUE THREAD

**Issue by ericchan1983** [SAP/ui5-webcomponents-react#2031]
**Describe the bug**
When the ObjectPage is the main content of the page (role="main") and there is another header (role="banner") above the ObjectPage.
The role="banner" is not suitable for the  ObjectPageTopHeader, but there is no way to remove the role property.
Thus the Axe tool report the issue below:
Banner landmark should not be contained in another landmark

**Isolated Example**
*Please provide an isolated example if possible by forking [this codesandbox](https://codesandbox.io/s/ui5-webcomponents-react-template-29l34).*

**To Reproduce**

**Expected behavior**
Use one suitable role instead of the "banner" in the above case.

**Screenshots**
![image](https://user-images.githubusercontent.com/1614528/130792624-fdc136c5-cda7-47eb-b629-db75b2fddfee.png)

**UI5 Web Components for React Information**
`@ui5/webcomponents` version:0.31.14
`@ui5/webcomponents-react` version:0.131.13
Operating System:macOS Big Sur 11.5
Browser:Chrome Version 92.0.4515.159 (Official Build) (x86_64)

**Additional context**

**Comment by MarcusNotheis** [comment]
Downported to `0.131.18`.
